drv/gimlet-seq-server/src/main.rs

```diff
@@ -1302,8 +1341,8 @@ cfg_if::cfg_if! {
             let (device, rail) = i2c_config::pmbus::vddcr_soc(i2c);
             let mut vddcr_soc = Raa229618::new(&device, rail);
 
-            retry_i2c_txn(|| vdd_vcore.turn_off())?;
-            retry_i2c_txn(|| vddcr_soc.turn_off())?;
+            retry_i2c_txn(I2cTxn::VCoreOff, || vdd_vcore.turn_off())?;
```

Comment:
Is there any reason to believe these two commands should be in a transaction together?  (same for `VCoreOn` / `SocOn` below)

Reply:
No, these don't need to be grouped together.